composers.rb

```diff
@@ -1,34 +1,75 @@
-beethoven = {
-  name: "Ludwig van Beethoven", year_born: 1770, year_died: 1827
-}
-kant = {
-  name: "Immanuel Kant", year_born: 1724, year_died: 1804
-}
-mozart = {
-  name: "Wolfgang Amadeus Mozart", year_born: 1756, year_died: 1791
-}
-
-
-def age(number_1, number_2) 
+# beethoven = {
+#   name: "Ludwig van Beethoven", year_born: 1770, year_died: 1827
+# }
+# kant = {
+#   name: "Immanuel Kant", year_born: 1724, year_died: 1804
+# }
+# mozart = {
+#   name: "Wolfgang Amadeus Mozart", year_born: 1756, year_died: 1791
+# }
+
+@composers = []
+@selected_composers = []
+require_relative './composer_class.rb'
+CSV_DELIMITER = ";"
+
+def analyse_composers
+  composers_csv = File.read("composers.csv")
```

Comment:
You could try using the CSV standard library for reading the csv (but do so in an other pull request)